elyra/metadata/schemasproviders.py

```diff
@@ -109,5 +114,4 @@ def get_schemas(self) -> List[Dict]:
 class ComponentRegistriesSchemas(ElyraSchemasProvider):
     """Returns schemas relative to Component Registries schemaspace."""
     def get_schemas(self) -> List[Dict]:
-        schemas = self.get_local_schemas_by_schemaspace(ComponentRegistries.COMPONENT_REGISTRIES_SCHEMASPACE_ID)
-        return schemas
+        return self.get_local_schemas_by_schemaspace(ComponentRegistries.COMPONENT_REGISTRIES_SCHEMASPACE_ID)
```

Comment:
Nit: Would you mind keeping this as two statements?  The reason (I prefer this) is for debugging since, with two statements, the `return` statement serves as a good breakpoint to check things. I just feel this introduces a good "maintainability point" that otherwise would need to take place at the `get_schemas()` call point (which may vary) or in `get_local_schemas_by_schemaspace` which includes stepping across the various schemaspace ids.